pandas/_libs/tslibs/np_datetime.pxd

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# cython: profile=False
+
+from numpy cimport int64_t, int32_t
```

Comment:
you could actually just call this module util i think

Reply:
I'd like to avoid name overlap with the existing `libs/src/util`module.  Also because in the dev branch I've ported src/util to a pure-cython (no C deps--> _much_ simpler setup.py) `tslibs.util`.  Don't want to get those mixed up.

My first choice is still the original `npy_dtime`, since `np_datetime` overlaps with the existing `libs/src/datetime/np_datetime` files.